src/MongoDB/Query.c

```diff
@@ -288,6 +288,7 @@ static bool php_phongo_query_init(php_phongo_query_t* intern, zval* filter, zval
 		}
 	}
 
+	PHONGO_QUERY_OPT_BOOL("allowDiskUse", options, "allowDiskUse")
```

Comment:
This PR is brought to you by `PHONGO_QUERY_OPT_BOOL`.

![](https://media.giphy.com/media/tXTqLBYNf0N7W/giphy.gif)